SourceX/DiabloUI/selhero.cpp

```diff
@@ -370,4 +375,53 @@ BOOL UiSelHeroMultDialog(
 	selhero_isMultiPlayer = true;
 	return UiSelHeroDialog(fninfo, fncreate, fnstats, fnremove, dlgresult, name);
 }
+
+const char *selhero_GenerateName(std::uint8_t hero_class)
+{
+	// TODO: Replace these with lore-appropriate names. Currently they're generic
+	// fantasy names.
+	static const char *const kNames[3][10] = {
+		{
+		    "Arth",

```

Comment:
Here is a better list of warrior names:
```
Aidan
Qarak
Born
Cathan
Halbu
Lenalas
Maximus
Vane
Myrdgar
Rothat
```

Aidan is the official name of the hero of D1 (as of D3), the rest are mentioned in various lore, some have short stories and others are mentioned by name only. I did a best attempt at picking characters that where most likly to have participated in D1.

Reply:
Done